File: Quiz.Android/lib/models/user_prize.dart
class UserPrize {
  int upId;
  int entityId;
  int prizeId;
  int userId;

  UserPrize({
    required this.upId,
    required this.entityId,
    required this.prizeId,
    required this.userId,
  });

  factory UserPrize.fromJson(Map<String, dynamic> json) {
    return UserPrize(
      upId: json['upId'],
      entityId: json['entityId'],
      prizeId: json['prizeId'],
      userId: json['userId'],
    );
  }
}

File: Quiz.Entity/Models/UserPrize.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Quiz.Entity.Models
{
    public class UserPrize
    {
        [Key]
        public int UpId { get; set; }

        public int EntityId { get; set; }

        [ForeignKey("Prizes")]
        public int PrizeId { get; set; }

        [ForeignKey("Users")]
        public int UserId { get; set; }

        public virtual Prize Prize { get; set; }
        public virtual User User { get; set; }
    }
}
